src/PkgDeps.jl

```diff
@@ -129,4 +152,28 @@ function find_downstream_dependencies(
     return downstream_dependencies
 end
 
+
+"""
+    users(pkg_name::String; pkg_registry_name::String=GENERAL_REGISTRY)
+
+Find all packages which use `pkg_name`. Use the `pkg_registry_name` to look up the UUID of `pkg_name`.
+
+# Arguments
+- `pkg_name::String`: Find users of this package
+
+# Keywords
+- `registry_name::String=GENERAL_REGISTRY`: Name of registry where `pkg_name` is active
+
+# Returns
+- `Array{String}`: All packages which are dependent on `pkg_name`
+"""
+function users(
+    pkg_name::String;
+    pkg_registry_name::String=GENERAL_REGISTRY,
```

Comment:
why does this default to only the `GENERAL_REGISTRY` and not all `reachable_registries()`?

Reply:
The `pkg_registry_name` is used to look up the UUID of the package. I did this because you can have the same name for a package in multiple registries, and it's done to avoid ambiguity. Ex; `Agents`, so you don't need to remember or look up the `UUID` for it, rather just pass in its name and where it lives.

I'm sure there is a better solution for this, but for now I think it's okay.

Reply:
yah, i think in future it might be good to match the Pkg.jl behaviour and have a `prompt`/`request`, i.e. something like
```julia
(jl_UA2e5L) pkg> add Agents
There are multiple registered `Agents` packages, choose one:
 > Registry: General - Path: https://github.com/JuliaDynamics/Agents.jl.git
   Registry: OtherRegistry - Path: https://gitlab.com/private/Agents.jl.git
```
Is there an issue for that?

Reply:
There currently is not one logged. I thought about doing something like this, but wasn't too sure how this package will be used in the future. I would not expect to live in a code base somewhere, but more as a REPL tool so this functionality would be good!